calibr8/__init__.py

```diff
@@ -6,7 +6,12 @@
     StudentTNoise,
     PoissonNoise,
 )
-from .contrib.base import (
+from .contrib.normal import (
+    BaseAsymmetricLogisticN,
+    BaseLogIndependentAsymmetricLogisticN,
+    BasePolynomialModelN,
```

Comment:
If we want to do it the same way `StudentT` was implemented, a `BaseModelN `is missing here.

Reply:
The `BaseModelT` is an artifact of the old class/inheritance design.
I'd like to get rid of it wihout breaking the API though. I'll check if we can add a `DeprecationWarning` for any code that uses it.